libs/domain/auth/logout/src/logout.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from 'lit';
+
+export const styles = css`
+  oryx-icon {
+    --oryx-icon-size: 24px;
+  }
```

Comment:
we can move the variable to the button selector as it will peers through component layers